Custom default settings OPENMETA-665

lefevre-fraser opened the pull request.

jonathonw commented:
Minor nitpick:  the folder name under %APPDATA% should be "OpenMETA", not "OpenMeta".  (This probably doesn't matter unless your appdata folder somehow ends up case-sensitive, but there are ways this can happen now on Windows, so might as well be thorough.)

lefevre-fraser commented:
@jonathonw 
I modified it to be OpenMETA,
I also realized that I wasn't creating that directory if it did not exist, which I added to the same commit
